torch/csrc/jit/codegen/cuda/lower_misaligned_vectorization.cpp

```diff
@@ -69,6 +69,45 @@ class MisalignedVectorizationModifier {
     }
   }
 
+  struct ReferenceTensors {
```

Comment:
nit: Could be a little confusing since we also have `ReferenceTensor` used in indexing. Not a big problem, though.